### DISCUSSION TITLE
Looking to add an overlay to Actor Collections pictures

### DISCUSSION THREAD

**Discussion by mdisturbed** [meisnate12/Plex-Meta-Manager#371] [Q&A]
I am recently new to Plex-Meta-Manager and love it so far, I have been building out my Movies.yml and want to add an overlay I will create to all Actor Collections that are created by the script below:

`Angelina Jolie:
    template: {name: people, person: 11701}`

The people portion of the script pulls the "head shot" from tmdb.  Any help would be huge.  Thank you in advance and keep up the awesome work.

**Comment by meisnate12** [comment]
collection overlays is not currently a function of PMM
